rules-reviewed/openshift/local-storage.windup.xml

```diff
@@ -67,9 +63,7 @@ It is critical to do an audit and find where your application uses local files,
             </when>
             <perform>
                 <hint title="Detected local file system paths" effort="1" category-id="potential">
-                    <message>Paths to local file(s) are sometimes used for **temporary files** or **caching**, but frequently for **persistent** data as well.
-                        File local storage is **transient** and could _disappear_ when the server instance is stopped/replaced/restarted. 
-                    No matter what you decide to use, it is critical to do an inventory and locate all possible local files storage accesses, and think about what you need to change in design in your application for Openshift environment.
+                    <message>Local files in containers are not permanent storage and can disappear when an instance of the application terminates. Avoid accessing local filesystems from applications running in OpenShift environments, and consider using a database or cloud-based storage system instead.
```

Comment:
This message is the same as above message for local-storage-00001. We should have unique message for every hint, otherwise it should be merged into one.